docs/source/ReleaseNotes.md

```diff
@@ -1,72 +1,104 @@
 # RELEASE NOTES
 
-* [v1.170](#v1-170)
-* [v1.169](#v1-169)
-* [v1.168](#v1-168)
-* [v1.167](#v1-167)
-* [v1.166](#v1-166)
-* [v1.165](#v1-165)
+- [v1.171.x](#v1171x)
+- [v1.170.x](#v1170x)
+- [v1.169.x](#v1169x)
+- [v1.168.x](#v1168x)
+- [v1.167.x](#v1167x)
+- [v1.166.x](#v1166x)
+- [v1.165.x](#v1165x)
```

Comment:
Should there be a Hyphen where the dots are for the bracketed versions? E.G (#v1-169-x) Tested the Docs https://docs.vualto.com/projects/task-engine/en/dev-branch/ReleaseNotes.html and clicking them doesn't interact in any way currently.

Reply:
Ehh thanks Pete... the MD extension I'm using keeps removing them 🙄 .